server/graphql/schema.js

```diff
@@ -14,12 +14,14 @@ const typeDefs = gql`
     addUser(username: String!, email: String!, password: String!): Auth
     updateUser(username: String!, email: String!, password: String!): User
     addMessage(sender: ID!, content: String!, thread: ID, location: ID!): Message
-    addChatroom(title: String!, tags: [ID], icon: String): Chatroom
+    addChatroom(title: String!, tagIds: [ID!], icon: String): Chatroom
   }
 
   type User {
     _id: ID
     username: String
+    activeChatrooms: [Chatroom]
```

Comment:
we should calculate active users for chatrooms themselves. `chatroom.js` has an `activeUsers` array that we can use for this